### DISCUSSION TITLE
Custom background colors for the boards

### DISCUSSION THREAD

**Discussion by papadeiv** [tableaunoir/tableaunoir#156] [Ideas]
Hi,

first off, I've been using tableaunoir for a month now, and I absolutely love it.
I was wondering if it would be possible in the near future to allow for multiple background colors for the boards? I personally have taught in the bast using dark grey and dark blue backgrounds but it would also be ideal to have the possibility of gradient-based backgrounds. Perhaps a user-defined color to be selected manually from a color-picker or wheel?

Keep up the good work.

**Comment by francoisschwarzentruber** [comment]
Dear Davide,

thank you for your kind message. Sure, having custom backgrounds sound relevant! Just few questions:
1. what are your motivations for having a dark grey or dark blue backgrounds? Is it for aesthetic reasons?
2. why do you need gradient-based backgrounds?
3. what kind of gradient do you need? linear? radial?

Have a nice day!

**Reply by papadeiv** [reply]
1. Yes and also for ease of categorisation if I teach different subjects, e.g. in the past I used to teach 3 courses in mathematics, physics and chemistry (secondary school) and I had three background for each of them.
2. Also for aesthetic purposes e.g. if I attach a short video of a simulation it helps to have non uniform background like white to grey.
3. I only used linear background with vertical alignment.
It would be just perfect to have them implemented; several examples of streamers in maths and physics using different coloured backgrounds can be found on YT and Twitch.

**Reply by francoisschwarzentruber** [reply]
Dear Davide,
thank you for your answer. I now understand your need I think. Now, you can select "Customboard" and choose a color, or a vertical gradient!
Does it work for you?

**Reply by papadeiv** [reply]
Yes, perfectly. Thank you very much; only the text magnet does not get the custome background color but instead retrieves to white.
![test-custom-bg](https://user-images.githubusercontent.com/52447965/116360342-19b9b780-a800-11eb-8dfa-271d1e5094a7.png)

**Reply by francoisschwarzentruber** [reply]
Yes indeed! Thank you, it works now!
(maybe not completely for gradient)

**Reply by papadeiv** [reply]
That'd be perfect, thank you very much.
